src/language-html/print/block.js

```diff
@@ -0,0 +1,123 @@
+import { group, hardline, indent } from "../../document/builders.js";
+import { printChildren } from "./children.js";
+
+let uid = 0;
+
+const settings = {
+  if: {
+    withExpression: true,
+    isFollowingBlock: false,
+    followingBlocks: ["else if", "else"],
+  },
+  "else if": {
```

Comment:
We need to normalize node name `else if`. Angular allows `else         if` current our implementation does not:

**Prettier pr-15606**
[Playground link](https://deploy-preview-15606--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEABAlgMwAQAoCU2wAOlAL7apwA2AznNo1nocGSADQgQAOM60WslABDAE5iIAdwAK4hEJQjqUkQE8hXAEZiRYANZwYAZREBbOABl0UOMkzL623QaPGeemwHNkMMQFc4LnozdF8AoJBab2o4AEV-CHh7R0iAK1oAD2MY+MTkpAc6SIBHfLgZSR5FEBFaAFpbOAATFs4QPxF0am8AYQgzMxFkWupqduioL1iAQRg-dC1-eBk4MWtbFOKuAAsYM2oAdR30eFoPMDhjBVP0ADdTtRGwWk0QO8CASShW2GMwMToPgzH7GGBqWJbJwgHiSeiHXQ8EawuD0MR3OxcGxomCVEReIZQyIeMRokYiKb+ajidqwmwwQ7oZowHbIAAcAAYuGI4GV0Dy8QThoVUlwYCItIzmaykAAmLj+egAFQliiK0LgZi0LVazUsFK8-nxcAAYhAxEN5t5ycsICAyGQgA)
<!-- prettier-ignore -->
```sh
--parser angular
```

**Input:**
<!-- prettier-ignore -->
```html
@if () {
} @else   if () {}
```

**Output:**
<!-- prettier-ignore -->
```html
Error: Unknown block name: else   if
    at is (https://deploy-preview-15606--prettier.netlify.app/lib/plugins/html.js:14:11151)
    at Object.ba [as print] (https://deploy-preview-15606--prettier.netlify.app/lib/plugins/html.js:14:12544)
    at Yn (https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:30:7727)
    at D (https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:30:7547)
    at s (https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:30:7355)
    at rt (https://deploy-preview-15606--prettier.netlify.app/lib/plugins/html.js:14:7858)
    at https://deploy-preview-15606--prettier.netlify.app/lib/plugins/html.js:14:8860
    at https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:26:1596
    at Vt.each (https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:26:1499)
    at Vt.map (https://deploy-preview-15606--prettier.netlify.app/lib/standalone.js:26:1576)
```

Reply:
Added `normalizeName` to avoid this case